src/chrome/content/rules/Catbox.moe.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+	Invalid Certificate:
+		www.catbox.moe
+		blog.catbox.moe
+-->
+<ruleset name="Catbox.moe">
+	<target host="catbox.moe" />
+	<target host="www.catbox.moe" />
+	<target host="files.catbox.moe" />
+	<target host="nofun.catbox.moe" />
+	<target host="qt.catbox.moe" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://www\.catbox\.moe/" to="https://catbox.moe/" />
```

Comment:
This constitutes a snapping redirect, which we try to avoid.  See https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects

Reply:
Snapping redirects are generally okay if they redirect "around" a broken URL, in this case avoiding `www` which has an invalid certificate. (This is implied by your link: "if https://foo.fm serves HTTPS correctly".) This is especially true for redirects between `^` and `www`, which should be very low-risk.

Reply:
@jeremyn Can you merge this one as well? (I ran travis again and fixed some problems FWIW)